src/jwt.cpp

```diff
@@ -109,10 +115,15 @@ Validator stringValidator(std::string&& name,
 
 std::string formatTime(std::time_t value) noexcept
 {
-    std::tm tmb;
     char buf[20];
-    gmtime_r(&value, &tmb);
-    auto res = std::strftime(buf, sizeof(buf), "%F %T", &tmb);
+#ifdef WIN32
+	std::tm * tmb=nullptr;
```

Comment:
Tab is used for indentation. Should be spaces. Also, there shouldn't be a space before the asterisk, because it forms a type along with std::tm. OTOH, there should be spaces around '='.